nixos/modules/services/web-servers/nginx/default.nix

```diff
@@ -40,7 +40,22 @@ let
     }
   ''));
 
-  configFile = pkgs.writeText "nginx.conf" ''
+  configFile = if cfg.reformatConfigFile then configFileFormatted else configFileUnformatted;
+
+  configFileFormatted = pkgs.runCommand "nginx.conf" {
+    inherit configFileUnformatted;
+    passAsFile = [ "configFileUnformatted" ];
```

Comment:
I do not think that these two lines are needed because the reference to `configFileUnformatted` in the multi-line string below is directly to the store path that `writeText` creates a few lines further down.

Reply:
@Baughn I can confirm the above two lines are not necessary.

Could you remove them? I would love to have this merged.